apps/protocol-frontend/src/components/DeleteContributionDialog.tsx

```diff
@@ -0,0 +1,82 @@
+import { useRef} from 'react';
+import { useUser} from '../contexts/UserContext';
+
+
+import {
+    AlertDialog,
+    AlertDialogBody,
+    AlertDialogFooter,
+    AlertDialogContent,
+    AlertDialogOverlay,
+    Button,
+    useDisclosure,
+  } from '@chakra-ui/react';
+
+
+const  DeleteContributionDialog = (props: any) => {
+
+    const {dialog, setDialog} = props;
+    
+    const { onClose } = useDisclosure()
+    const cancelRef = useRef<HTMLButtonElement>(null);
+    
+    const { deleteContribution } = useUser();
+
+    const DeleteContribution = async (contribution_id: string) => { 
+      const y = await deleteContribution(contribution_id);
+      
+    };
+
+    const onDelete = (onConfirm: boolean, contribution_id: string) => {
+      if (onConfirm){
+        setDialog({ ...dialog, 
+          isOpen: false,
+         })
+        DeleteContribution(contribution_id)
+      }
+    }
+   
+    return (
+        
+        <AlertDialog
+          isOpen={dialog.isOpen}
+          leastDestructiveRef={cancelRef}
+          onClose={onClose}
+  
+        >
+          <AlertDialogOverlay >
+            <AlertDialogContent bgColor="white" color="gray.800">
+            
+              <AlertDialogBody mt='20px'>
+                {dialog.title}
+              </AlertDialogBody>
+  
+              <AlertDialogFooter>
+                <Button 
+                 ref={cancelRef} 
+                 onClick={() => setDialog({ ...dialog, isOpen: false })} 
+                 >
+                  Cancel
+                </Button>
+
+                <Button 
+                 colorScheme='red' 
+                 color="brand.primary.600"
```

Comment:
we need to remove the `color` and the `backgroundColor` -- they're overriding the `colorScheme`

Reply:
I just removed those. Thanks!